addons/l10n_co/data/account_tax_template.xml

```diff
@@ -131,13 +131,16 @@
         <field name="tax_group_id" ref="tax_group_r_iva_285"/>
         <field name="invoice_repartition_line_ids" eval="[(5, 0, 0),
             (0,0, {'repartition_type': 'base'}),
-            (0,0, {'repartition_type': 'tax'}),
+            (0,0, {
+                'repartition_type': 'tax',
+                'account_id': ref('co_puc_236700''),
```

Comment:
Hi @aliyatastemirova I have an error on this sintaxis, can you help me to switch it to 'co_puc_236700'.